lib/ansible/modules/database/postgresql/postgresql_table.py

```diff
@@ -260,12 +265,19 @@ def get_info(self):
 
     def __exists_in_db(self):
         """Check table exists and refresh info"""
+        if "." in self.name:
+            schema = self.name.split('.')[-2]
```

Comment:
Not a massive change but I'd suggest doing the split like this

```suggestion
            schema, tblname = self.name.split('.', 2)
```

Reply:
It can contain the db like (will be with 2 dots)
```
>>> table = 'db.schema.tblname'
>>> s, t = table.split('.', 2)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: too many values to unpack (expected 2)
```

Reply:
Ah, fair enough.